src/react-components/avatar-editor.js

```diff
@@ -108,6 +109,27 @@ export default class AvatarEditor extends Component {
       URL.revokeObjectURL(gltfUrl);
 
       const { content, body } = parser.extensions.KHR_binary_glTF;
+
+      // Inject hubs components on upload. Used to create base avatar
+      // const gltf = parser.json;
```

Comment:
Uncomment or delete?

Reply:
There is manual work involved in updating the "base" avatar and the simplest way I know how to do it is to just hack up the form. I could extract this elsewhere but given its temporary nature it doesn't seem worth the effort, so its relegated to a comment based tool heh.